src/build/browser/browser-static/bootstrap.js

```diff
@@ -134,11 +92,39 @@ function bootstrap(initialImport, target) {
 
 	var loaded = false;
 	w._continueLoad = function() {
-		if (!loaded) {
-			loaded = true;
-			var el = d.createElement('script');
-			el.src = target + '.js';
-			d.getElementsByTagName('head')[0].appendChild(el);
+		var doThings = function() {
+			if (!loaded) {
+				loaded = true;
+				// Include the game code
+				var el = d.createElement('script');
+				el.src = target + '.js';
+				d.getElementsByTagName('head')[0].appendChild(el);
+			}
+		};
+
+		var _continueLoadDefer;
+
+		w.addEventListener('message', function(event) {
+			if (event.data === 'partialLoadContinue') {
+				if (_continueLoadDefer) {
+					_continueLoadDefer.resolve();
+					_continueLoadDefer = undefined;
+				}
+			}
+		});
+		// Preload suggestions and then tell the parent bootstrapping is complete
+		jsio.__env.preloadModules(function() {
+			w.parent.postMessage('bootstrapping', '*');
+		});
+
+		var partialLoadKey = jsio.__env.getNamespace('partialLoad');
+		if (localStorage && localStorage.getItem(partialLoadKey)) {
+			localStorage.removeItem(partialLoadKey);
```

Comment:
Is `jsio` namespacing only used for `localStorage`?  If so, we should just solve the `localStorage` problem separately -- we can wrap the `localStorage` API in devkit on a per-app basis